Automattic/wp-calypso
# Head: generate touch icon tags from sizes array

[As suggested by](https://github.com/Automattic/wp-calypso/pull/21424#discussion_r165169763) @samouri we now generate parts of the favicon link tags instead of writing each of them manually.

### How to test

Make sure any `<link rel="apple-touch-icon" />` tags inside of `<head />` match the current output on WordPress.com.
